acceptance/Rakefile

```diff
@@ -233,16 +233,19 @@ def sha
   ENV['SHA']
 end
 
+# Return the host config file for beaker to use
+# If TEST_TARGET is set, pass it to beaker-hostgenerator to generate a file
+# else, assume CONFIG is set to the path of an existing host config file
 def config
-  if (ENV['TEST_TARGET']) && (not ENV['CONFIG'])
+  if (ENV['TEST_TARGET'])
     if !ENV['TEST_TARGET'].start_with?('redhat7-64m')
       ENV['TEST_TARGET'] = "redhat7-64m-#{ENV['TEST_TARGET']}"
-      puts "Adding EL7 master to ENV['TEST_TARGET']. Value is now '#{ENV['TEST_TARGET']}'"
+      puts "Adding Redhat 7 master to ENV['TEST_TARGET']. Value is now '#{ENV['TEST_TARGET']}'"
     end
     cli = BeakerHostGenerator::CLI.new([ENV['TEST_TARGET'], '--disable-default-role'])
     ENV['CONFIG'] = "tmp/#{ENV['TEST_TARGET']}-#{SecureRandom.uuid}.yaml"
     FileUtils.mkdir_p('tmp')
-    File.open(config, 'w') do |fh|
+    File.open(ENV['CONFIG'], 'w') do |fh|
```

Comment:
In this case, the ENV is set two lines above, and this is the logic to write that desired file out. So the intent is that the file doesn't exist.

But to riff on that question: there's a good concern about error handling in general, e.g. when a developer specifies CONFIG on the invocation line for local testing. I assume beaker will fail dramatically in any of those cases. It would be slick if, perhaps as part of refactoring/commonizing this logic across projects/branches (rototiller is my new dream for a brighter future :)), if we had graceful handling of such scenarios with helpful error messages.